docs/network/network-binding-plugin.md

```diff
@@ -505,3 +505,15 @@ derived from virtual machines using the plugin.
 For a network binding plugin to support compute resource overhead, the `computeResourceOverhead` field
 must be specified in the Kubevirt CR.
 See the user-guide network binding plugin [section](https://kubevirt.io/user-guide/network/network_binding_plugins/#register) on how to define it.
+
+## Network plugin custom sockets
```

Comment:
@oshoval Sorry but the use case of "custom socket" is unclear to me from the proposed documentation.
Why would a plugin author need custom socket, in addition to the hook socket? I would write that in some corner cases when there is a need to create another communication channel between the virtualization stack in compute container and the network plugin on the sidecar container side, in that case please use another depth i.e. `/var/run/kubevirt-hooks/plugin-libvirt-comm/comm.sock`. Using the CONTAINER_NAME the hook author can feed the virtualization stack with the relevant path for the plugin-virt communication.